stable/cockroachdb/Chart.yaml

```diff
@@ -1,7 +1,8 @@
 name: cockroachdb
 home: https://www.cockroachlabs.com
 version: 0.2.1
-description: CockroachDB Helm chart for Kubernetes.
+description: CockroachDB is a scalable, survivable, strongly-consistent SQL database.
+icon: https://www.cockroachlabs.com/wp-content/themes/cockroach-labs/images/CL_Logo_Horizontal.png
```

Comment:
https://raw.githubusercontent.com/cockroachdb/cockroach/master/docs/media/cockroach_db.png would probably be better, since it's for the database rather than the company backing it.